hedera-node/src/main/java/com/hedera/services/sigs/metadata/lookups/BackedAccountLookup.java

```diff
@@ -36,19 +36,10 @@ public BackedAccountLookup(BackingStore<AccountID, MerkleAccount> accounts) {
 
 	@Override
 	public SafeLookupResult<AccountSigningMetadata> safeLookup(AccountID id) {
-		return lookup(id, false);
-	}
-
-	@Override
-	public SafeLookupResult<AccountSigningMetadata> pureSafeLookup(AccountID id) {
-		return lookup(id, true);
-	}
-
-	private SafeLookupResult<AccountSigningMetadata> lookup(AccountID id, boolean usePure) {
 		if (!accounts.contains(id)) {
 			return SafeLookupResult.failure(MISSING_ACCOUNT);
 		}
-		var account = usePure ? accounts.getUnsafeRef(id) : accounts.getRef(id);
+		var account = accounts.getUnsafeRef(id);
```

Comment:
nit: Should we rename the function as we are call `getUnsafeRef`?

Reply:
Changed to `getImmutableRef()` 👍